### PR TITLE
Software Suggestion | DecSync

### PR DESCRIPTION
**Pull request by moritztk**
## Description

Resolves: #1416 

#### Check List 

- [X] I have read and understand [the contributing guidelines](https://github.com/privacytoolsIO/privacytools.io/blob/master/.github/CONTRIBUTING.md).

- [X] The project is [Free Libre](https://en.wikipedia.org/wiki/Free_software) and/or [Open Source](https://en.wikipedia.org/wiki/Open-source_software) Software

* Netlify preview for the mainly edited page: https://deploy-preview-1530--privacytools-io.netlify.com/software/calendar-contacts/#calendar_contacts

* Code repository of the project (if applicable): https://github.com/39aldo39/DecSync

### CODE REVIEW

**Comment by moritztk** [comment]
Hi Mikaela

> https://github.com/39aldo39/DecSync/pulse/monthly - the project isn't very active according to GitHub's monthly pulse.

This is the main repo, just describing the project in docs. The repos containing the apps are more active: [Android](https://github.com/39aldo39/DecSyncCC/pulse/monthly) and [Linux](https://github.com/39aldo39/Evolution-DecSync/pulse/monthly).

> Is productivity tools the correct section? I understand that it belongs there too, but I wonder if people would be looking for it in calendar/contacts sync? 

TBH I am not quite sure about that myself and would be happy about suggestions. 

>  I wonder if there should be an issue about merging the two pages.

I don't quite understand what you mean by that, maybe I am missing some context?

**Comment by moritztk** [comment]
Added a link to syncthing

**Comment by moritztk** [comment]
I moved the link from productivity to cal/con, and somehow accidentaly dismissed your review.

**Comment by Mikaela** [comment]
GitHub tells me 0 changed files and reviews are automatically dismissed as something changes, probably in case it fixes old change request or turns from approved to unapproved.

**Comment by moritztk** [comment]
@Mikaela the change should show again, sorry for the confusion

File: _includes/sections/calendar-contacts-sync.html [hunk 1]
```diff
@@ -60,4 +60,8 @@ <h3>Worth Mentioning</h3>
   <li>
     <a href="../cloud">cloud backups</a> - Consider regularly exporting your calendar and or contacts and backing them up on a separate storage drive or uploading them to cloud storage (ideally after <a href="../encryption-tools/">encrypting</a> them).
   </li>
+  
+  <li>
+    <a href="https://github.com/39aldo39/DecSync">DecSync</a> - DecSync can be used to synchronize RSS, contacts and calendars.  This can be done without a server by using for example <a href="https://syncthing.net/">Syncthing</a>.
```
Comment (nitrohorse) [inline _includes/sections/calendar-contacts-sync.html]:
```suggestion
    <a href="https://github.com/39aldo39/DecSync">DecSync</a> - DecSync can be used to synchronize RSS, contacts, and calendars without a server by using file synchronization software such as <a href="https://syncthing.net/">Syncthing</a>.
```

**Comment by moritztk** [comment]
@nitrohorse commited your suggestion, I agree it's clearer.
Somehow that dismissed @Mikaela's review again.

**Pull request merged**